moveit_experimental/jog_arm/README.md

```diff
@@ -0,0 +1 @@
+http://wiki.ros.org/jog_arm
```

Comment:
Is the intention to move this package from https://github.com/UTNuclearRoboticsPublic/jog_arm.git, listed on this wiki page, to here?

can you create a tutorial for this package in moveit_tutorials so that people can easily use and contribute to your effort?